tools/scripts/fuzz_minimize_crash.sh

```diff
@@ -1,71 +1,100 @@
 #!/bin/bash
 
-set -exuo pipefail
-
-if (( $# != 1 )); then
-    echo "Illegal number of parameters. Need a single file to minimize"
+set -euo pipefail
+cd "$(dirname "$0")"
+cd "../.."
+# we're now at the root of the repo.
+
+if [ "$#" -lt 2 ]; then
+cat <<EOF
+usage:
+  $0 <target> <crasher> [<options>]
+
+example target:
+  fuzz_dash_e
+  fuzz_doc_symbols
+  fuzz_hover
+
+example crasher:
+  fuzz_crashers/original/crash-[...]
+
+example options:
+  --stress-incremental-resolver
+EOF
+exit 1
 fi
 
-FUZZ_ARG="--stress-incremental-resolver" # to run incremental resolver
+target="$1"
+shift
+crasher="$1"
+shift
 
-dir="$( dirname "${BASH_SOURCE[0]}" )"
+mkdir -p fuzz_crashers/fixed/min fuzz_crashers/fixed/original fuzz_crashers/min
 
-mkdir -p "${dir}/../../fuzz_crashers/fixed/min/" "${dir}/../../fuzz_crashers/fixed/original/"
+crasher_name="$(basename "$crasher")"
+output_file="fuzz_crashers/min/$crasher_name"
+done_file="$output_file.done"
```

Comment:
Does this work without the `${}` around `output_file`?

Reply:
I don't think var names can have . in them (per syntax highlighting). Do you want me to keep {} just as a style thing?

Reply:
I just wanted to make sure it works. If it works, then that's fine.